negotiation/message/contract-agreement-message.json

```diff
@@ -4,15 +4,21 @@
   "dspace:providerPid": "urn:uuid:a343fcbf-99fc-4ce8-8e9b-148c97605aab",
   "dspace:consumerPid": "urn:uuid:32541fe6-c580-409e-85a8-8a9a32fbe833",
   "dspace:agreement": {
-    "@id": "e8dc8655-44c2-46ef-b701-4cffdc2faa44",
+    "@id": "urn:uuid:e8dc8655-44c2-46ef-b701-4cffdc2faa44",
     "@type": "odrl:Agreement",
-    "dspace:consumerId": "...",
-    "dspace:providerId": "...",
+    "odrl:target": "urn:uuid:3dd1add4-4d2d-569e-d634-8394a8836d23",
     "dspace:timestamp": "2023-01-01T01:00:00Z",
-    "odrl:permission": [
-      {
-        ...
-      }
-    ]
-  }
+    "odrl:assigner": "urn:tsdshhs636378",
```

Comment:
What about #92? We decided to use `dspace:consumerId` and `dspace:providerId` some time ago. 

(including @jimmarino in the conversation)